blocks/card-list-block/features/card-list/default.jsx

```diff
@@ -208,12 +208,16 @@ const CardListItems = (props) => {
 								</Heading>
 								<Attribution>
 									{hasAuthor ? (
-										<>
-											<span>{phrases.t("global.by-text")}</span> <span>{bylineNodes}</span>
+										<div className={`${BLOCK_CLASS_NAME}__byline-container`}>
+											<span>
+												{phrases.t("global.by-text")} {bylineNodes}
```

Comment:
These still need to be in their own `span`s, otherwise they're in the wrong order for RTL. Though it then brings up an issue with how to handle the space between them. The best way would be to give the span around `phrases.t("global.by-text")` a class, then add `margin-inline-end: some-global-spacing-var` as a property for it. Not sure which spacing value would look best.